src/components/ordersPage/content.js

```diff
@@ -4,21 +4,142 @@
+        let all_orders = $('<div id="all-orders"></div>');
+
+        let orders = $('<div  id="orders"></div>');
+        let del_orders = $('<div  id="delivered-orders"></div>');
+        let header_order = $('<div class="header-order"><span data-textid="order-header-order"></span></div>');
+        let header_del = $('<div class="header-delivered"><span data-textid="order-header-delivered"></span></div>');
+        let cont_orders = $('<div  id="orders-container"></div>');
+        let cont_del_orders = $('<div id="delivered-orders-container" ></div>');
+
+        $(all_orders).append(orders);
+        $(all_orders).append(del_orders);
+        $(del_orders).append(header_del);
+        $(orders).append(header_order);
+        $(del_orders).append(cont_del_orders);
+        $(orders).append(cont_orders);
+
+        for (let i = 0; i < this.orders.length; i++){
+            $(cont_orders).append(this.createOrderCard( this.orders[i], i));
+        }
+         $(cont_orders).on("drop", (ev) => {
+            ev.preventDefault();
+            var data = ev.originalEvent.dataTransfer.getData("text");
+            ev.target.appendChild(document.getElementById(data));
+             ev.stopPropagation();
+        });
+
+        $(cont_orders).on("dragover", (ev) => {
+           ev.preventDefault();
+           ev.stopPropagation();
+        });
+
+        $(cont_del_orders).on("drop", (ev) => {
+            ev.preventDefault();
+            var data = ev.originalEvent.dataTransfer.getData("text");
+            ev.target.appendChild(document.getElementById(data));
+            ev.stopPropagation();
+        });
+
+        $(cont_del_orders).on("dragover", (ev) => {
+           ev.preventDefault();
+           ev.stopPropagation();
+        });
+
+        return all_orders;
+    }
+    
+    
+    createOrderCard = (order_cont,id) => {
+        let cont_card = $('<div draggable="true" id="' + id + '" class="container-card"></div>');
+        let card = $('<div class="order-card"></div>');
+        let content_card = $('<div class="content-card"></div>');
+
+
+        $(cont_card).on("dragstart", (ev) => {
+            console.log(ev.originalEvent.dataTransfer);
+            ev.originalEvent.dataTransfer.setData("text", ev.target.id);
+            ev.stopPropagation();
+        });
+
+
+
+        let table = $('<div class="container-table">'+
+                        '<div class="txt-table"><span data-textid="order-table"></span>:</div>'+
+                        '<div class="table"> ' + order_cont.table + '</div>'+
+                    '</div>');
+
+        let time = $('<div class="container-time">'+
+                        '<div class="txt-time"><span data-textid="order-time"></span>:</div>'+
+                        '<div class="time"> ' + order_cont.time + '</div>'+
+                    '</div>');
+
+        let order = $('<div class="container-order">'+'<div class="txt-order"><span data-textid="order-order"></span>:</div>')
```

Comment:
Missing a closing div
```suggestion
        let order = $('<div class="container-order"><div class="txt-order"><span data-textid="order-order"></span>:</div></div>')
```